### PR TITLE
[DOCS-3108] Add AWS Config Guide for Cloud SIEM

### PR DESCRIPTION
**Pull request by maycmlee**
### What does this PR do?

- Adds new AWS Configuration Guide for Cloud SIEM doc.
- Adds links to the guide in the Cloud SIEM Getting Started page.
- Adds link to the guide in the Cloud SIEM Guide page.

### Motivation

DOCS-3108

### Preview

https://docs-staging.datadoghq.com/may/aws-config-cloud-siem/security_platform/guide/aws-config-guide-for-cloud-siem/

https://docs-staging.datadoghq.com/may/aws-config-cloud-siem/security_platform/cloud_siem/getting_started/

https://docs-staging.datadoghq.com/may/aws-config-cloud-siem/security_platform/guide/

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by jmassey-datadog — APPROVED** [review]

**Comment by jmassey-datadog** [comment]
Can we do a find and replace for Cloudtrail to CloudTrail?

**Comment by jmassey-datadog** [comment]
Can we also reference the AWS Investigation dashboard in the "Use Cloud SIEM to triage Security Signals" section? It includes a lot of useful information to users and will be immediately useful to users whereas the customer will have to wait a little while for a security signal to be generated. 

Here is the relative URL for it. dash/integration/30459/aws-cloudtrail

**Comment by maycmlee** [comment]
Hey @jmassey-datadog! I should have gotten all the Cloudtrail -> Cloud Trail changes. Also added the info AWS investigation dashboard. Let me know if that works.

File: content/en/security_platform/guide/aws-config-guide-for-cloud-siem.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+## Set up AWS integration using CloudFormation
+
+1. Go to Datadog's [AWS integration tile][2] to install the integration.
+2. Click **Automatically Using CloudFormation**. If there is already an AWS account set up, click **Add Another Account** first.
+3. For Cloud SIEM, log management needs to be integrated, so select **Log Management**. This sets up the Datadog Lambda Forwarder to be used later for sending AWS CloudTrail logs to Datadog.
+4. Select the AWS Region where the CloudFormation stack will be launched. 
+5. Select or create the Datadog API Key used to send data from your AWS account to Datadog.
+6. Click **Launch CloudFormation Template**. This opens the AWS Console and loads the CloudFormation stack with the parameters filled in based on your selections in the prior Datadog form. 
+
+    **Note:** The DatadogAppKey parameter enables the CloudFormation stack to make API calls to Datadog to add and edit the Datadog configuration for this AWS account. The key is automatically generated and tied to your Datadog account.
```
Comment (cswatt) [inline content/en/security_platform/guide/aws-config-guide-for-cloud-siem.md:45]:
```suggestion
    **Note:** The `DatadogAppKey` parameter enables the CloudFormation stack to make API calls to Datadog to add and edit the Datadog configuration for this AWS account. The key is automatically generated and tied to your Datadog account. 
```

File: content/en/security_platform/guide/aws-config-guide-for-cloud-siem.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+3. For Cloud SIEM, log management needs to be integrated, so select **Log Management**. This sets up the Datadog Lambda Forwarder to be used later for sending AWS CloudTrail logs to Datadog.
+4. Select the AWS Region where the CloudFormation stack will be launched. 
+5. Select or create the Datadog API Key used to send data from your AWS account to Datadog.
+6. Click **Launch CloudFormation Template**. This opens the AWS Console and loads the CloudFormation stack with the parameters filled in based on your selections in the prior Datadog form. 
+
+    **Note:** The DatadogAppKey parameter enables the CloudFormation stack to make API calls to Datadog to add and edit the Datadog configuration for this AWS account. The key is automatically generated and tied to your Datadog account. 
+
+7. Check the required boxes from AWS and click **Create stack**.
+8. After the Stack is created, go back to the AWS integration tile in Datadog and find the box for the new account you created. Click **Refresh to Check Status** to see a success message at the top of the page, along with the new account visible on the page with the relevant details.
```
Comment (cswatt) [inline content/en/security_platform/guide/aws-config-guide-for-cloud-siem.md:48]:
```suggestion
8. After the CloudFormation stack is created, go back to the AWS integration tile in Datadog and find the box for the new account you created. Click **Refresh to Check Status** to see a success message at the top of the page, along with the new account visible on the page with the relevant details.
```

**Review by cswatt — CHANGES_REQUESTED** [review]
just small nits!

**Review by cswatt — APPROVED** [review]

**Pull request merged**